linetools/scripts/lt_continuumfit.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+
+"""
+Enable continuum fitting of an input spectrum
+"""
+import pdb
+import sys
+
+
+# Script to run continuum fitting in linetools
+def main(*args, **kwargs):
+    """ Runs the continuum fitter
+    """
+    import argparse
+
+    parser = argparse.ArgumentParser(description='Parse for Continuum Fitting')
+    parser.add_argument("file", type=str, help="Spectral file")
+    parser.add_argument("outfil", type=str, help="Output file [can be the same]")
+    parser.add_argument("-zsys", type=float, help="Redshift of the Source")
+    parser.add_argument("-wchunk", type=float, help="Width of a 'chunk' (Ang)")
```

Comment:
I suggest we use the convention that a single '-' should have a single character flag (i.e. '-h'), and the double '--' should be used for multiple character flags. So these would be changed to '--zsys' and `--wchunk`.